models/system/bert-base-cased/spec.yaml

```diff
@@ -37,5 +37,9 @@ tags:
   - Standard_ND40rs_v2
   - Standard_ND96asr_v4
   license: apache-2.0
+  model_specific_defaults:
+    apply_deepspeed: 'true'
+    apply_lora: 'true'
+    apply_ort: 'true'
   task: fill-mask
 version: 7
```

Comment:
@shashidhar-k Is it possible to update the model tags w/o bumping the version number?

Reply:
@elesel is working on this.

Reply:
Ok. thanks! I will update the model versions for the time being